libbeat/outputs/mode/balance.go

```diff
@@ -282,19 +279,13 @@ func (m *LoadBalancerMode) forwardEvent(
 	msg eventsMessage,
 ) bool {
 	if msg.attemptsLeft < 0 {
-		select {
-		case ch <- msg:
-			return true
-		case <-m.done: // shutdown
-			return false
-		}
+		ch <- msg
+		return true
 	} else {
 		for ; msg.attemptsLeft > 0; msg.attemptsLeft-- {
 			select {
 			case ch <- msg:
 				return true
-			case <-m.done: // shutdown
-				return false
 			case <-time.After(m.timeout):
```

Comment:
same here, why remove Done channel. The timeout is quite large by default, if ch is blocking users will 'perceive' the waiting time as 'blocked'